### PR TITLE
Review and Resubmit

### PR DESCRIPTION
**Pull request by katyhuff**
This PR contains the minor revisions suggested by the reviewers from Advances in Engineering Software. This doesn't complete all suggested revisions. A few less trivial revisions remain to adequately satisfy those needs. 

Each of the recommendations was addressed with a separate commit message, where possible. The recommendations addressed include:

## Reviewer 1
- Line 15: You indicate "current tools are typically privately distributed" - such as? Reviewer can think of ORION and possibly COSI - are there others? (Indicate examples if possible)
- Line 17: "inflexible architectures" - authors  explain this a little more later in the paper, but consider expanding your meaning briefly here.
- Line 67: toxicity is best described as radiotoxicity; mined uranium is mined uranium requirements. Meanwhile it's probably useful to express these in terms of per unit energy.
- Line 102: "nuclear technologies" - probably better described as nuclear technologies and facility types.
- Line 160: What platform does CAFCA rely upon? (Simulink? PowerSim? VISION for example relies on PowerSim; even when universities are able to get their hands on it, modification is indeed difficult because of the proprietary software requirement).
- Line 161: I think it's important to note here in this context that Cyclus is based on C++, where development tools are freely available and an open, well-documented standard.
- Line 227: What kind of nonproliferation applications is this significant to? What kinds of questions *can* Cyclus answer? This need not be terribly explicit, but I would suggest giving some further detail rather than alluding to it and then leaving it unspecified.
- Line 270: "Resource computing infrastructures" - this term might warrant further definition. (I assume you mean systems like cloud computing and batch scheduling computing clusters.)
- Line 278: proof-of-principle
- Line 339: "is superior to the system dynamic approach." I feel like a claim is being made here about superiority without much substantiation or expression of what the merit criteria is. This paragraph could use some further unpacking.
- Line 353: "The API abstracts away unnecessary detail" => suggest phrasing as "abstracts away details unnecessary to the implementation of the archetype-specific behavior"
- Figure 3 could be a bit bigger.
- Line 379: Directed acyclic graph: I suspect most readers will not be familiar with graph theory. A brief explanation and unpacking of the relevant context would be useful here.
- Line 398: "However, some questions require resolution at the level of individual facilities and materials." Such as? No need to go into explicit detail here, but given that you're building a case for flexible scoping, it would be helpful to produce a few examples that help to make your case. (For example, detailed isotopic analysis for thermal recycle vs. recipes. Nonproliferation applications. Etc.)
- Line 649: "because the new version is almost always wrong." This wording seems dangerously unclear. I think the intent is to say that deviation from prior, blessed results is almost always wrong (unless the change is expected, obviously). What is "wrong" are departures in results from a baseline.
- Line 785: Define heavy metal (and its abbreviation) for non-nuclear engineers.

## Reviewer 2

- Section 1.2 Line 127 change facilitates to facilitate
- Section 2.5.1 Line 616 Do the authors  mean tool git or tool kit? In English English git is a pejorative term and will cause some amusement!
- Section 2.5.3 Line 670 correct spelling of automated
- Section 3.1.2 line 754 Should the table being referenced be Table 3 rather than Table 1?

### CODE REVIEW

**Comment by gidden** [comment]
I have a PR into your branch with some suggested edits. Some other thoughts:

Reviewer 1:
Line 398: I feel like "to meaningfully optimize the cost and environmental impact" is a bit strong (and we know that some people don't think these tools should be used for "cost optimization"). Maybe we could just say "meaningful models of spent nuclear fuel storage transport, and disposal strategies, require representation of discrete casks and their varying isotopic compositions."

Reviewer 2:
I agree that most of the suggestions would require nontrivial rewrites or additional work. I'm happy to look at whatever you decide this week regarding those.

I think this is largely ready to go!

**Comment by katyhuff** [comment]
On Mon, Nov 30, 2015 at 12:35 AM, Matthew Gidden <notifications@github.com>
wrote:

> I have a PR into your branch with some suggested edits. Some other
> thoughts:
>
> Reviewer 1:
> Line 398: I feel like "to meaningfully optimize the cost and environmental
> impact" is a bit strong (and we know that some people don't think these
> tools should be used for "cost optimization"). Maybe we could just say
> "meaningful models of spent nuclear fuel storage transport, and disposal
> strategies, require representation of discrete casks and their varying
> isotopic compositions."
>

Good call.

> Reviewer 2:
> I agree that most of the suggestions would require nontrivial rewrites or
> additional work. I'm happy to look at whatever you decide this week
> regarding those.
>
> Thanks. We'll see what we can do.


> I think this is largely ready to go!
>

Yay!

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/fundamentals-paper/pull/141#issuecomment-160554545>
> .
>



-- 
http://katyhuff.github.com

File: fundamentals/impl.tex [hunk 1]
```diff
@@ -219,7 +220,11 @@ \subsubsection{Regions, Institutions, and Facilities}
 \Cyclus implements a \gls{RIF} relationship through a
 parent-child hierarchy where
 regions are the parents of institutions which are, in turn, the parents of
-facilities. In other words, \gls{RIF} hierarchies form a directed acyclic graph (DAG),
+facilities. In other words, \gls{RIF} hierarchies form a \gls{DAG}\footnote{
```
Comment (gonuke) [inline fundamentals/impl.tex]: alternatively, we could remove reference to a DAG - is that detail important?

**Comment by gonuke** [comment]
I PR'ed 2 typos, but otherwise am happy with all the changes.  Thanks @katyhuff !

**Comment by katyhuff** [comment]
Pulling these minor changes. Resolutions to the broader questions upcoming in a separate PR.

**Pull request merged**